barretenberg/cpp/src/barretenberg/common/ref_array.hpp

```diff
@@ -25,15 +25,11 @@ template <typename T, std::size_t N> class RefArray {
             storage[i++] = &elem;
         }
     }
-    RefArray(std::initializer_list<T&> init)
+    template <typename... Ts> RefArray(T& ref, Ts&... rest)
     {
-        if (init.size() != N) {
-            throw std::invalid_argument("Initializer list size does not match RefArray size");
-        }
-        std::size_t i = 0;
-        for (auto& elem : init) {
-            storage[i++] = &elem;
-        }
+        storage[0] = &ref;
```

Comment:
This wasn't actually working before, can't have an initializer list of references for the same reason you can't have a vector of references